ci/mkpipeline.py

```diff
@@ -89,6 +89,16 @@ def main() -> int:
         default=os.getenv("CI_PRIORITY", 0),
     )
     parser.add_argument("pipeline", type=str)
+    parser.add_argument(
+        "--bazel",
+        default=ui.env_is_truthy("CI_BAZEL_BUILD"),
```

Comment:
```suggestion
        default=ui.env_is_truthy("CI_BAZEL_BUILD", "1"),
```
I believe this would fix the other pipelines too.